src/components/modals/CompareDialogsModal.tsx

```diff
@@ -133,8 +180,8 @@ class CompareDialogsModal extends React.Component<Props, ComponentState> {
         }
 
         let logDialogId: string | undefined
-        const activityMap = new Map<string, BB.Activity[]>()
-        const rankMap = new Map<string, number | undefined>()
+        const activityMap = {}
```

Comment:
I'm supersized this worked. Isn't it supposed to be typed as `{[key: string]: BB.Activity[]}`  and same for other.
Assigning blank object seems weird. The type information on map provided more guarantees

Reply:
How else would you initialize an empty instance?